mobile-widgets/qml/DiveDetailsView.qml

```diff
@@ -218,20 +215,89 @@ Item {
 		rowSpacing: Kirigami.Units.smallSpacing * 2
 		columnSpacing: Kirigami.Units.smallSpacing
 
-		QMLProfile {
-			id: qmlProfile
-			visible: !noDive
+		Rectangle {
 			Layout.fillWidth: true
 			Layout.preferredHeight: Layout.minimumHeight
 			Layout.minimumHeight: width * 0.75
 			Layout.columnSpan: 3
-			clip: false
-			Rectangle {
-				color: "transparent"
-				opacity: 0.6
-				border.width: 1
-				border.color: subsurfaceTheme.primaryColor
+			clip: true
+
+			QMLProfile {
+				id: qmlProfile
+				visible: !noDive
 				anchors.fill: parent
+				clip: true
+				property real lastScale: 1.0 // final scale at the end of previous pinch
+				Rectangle {
+					color: "transparent"
+					opacity: 0.6
+					border.width: 1
+					border.color: subsurfaceTheme.primaryColor
+					anchors.fill: parent
+				}
+				PinchArea {
```

Comment:
I just learned what "pinching" is in mobile UIs. Interesting.

Reply:
Ha. I don't believe this for one second.
Unless, of course, you still have a flip phone.
Given some of your other comments earlier that does seem like a possibility :-P

Reply:
I knew the "motion", I didn't know what it was called. Off-topic: I detest when Google maps turns the north somewhere else because I'm not doing a perfectly vertical motion. Going to search how to prevent that right now...

Reply:
there are so many small UI issues to be fixed.
not sure that would make the top 10 of my annoyances - but every bit of help is welcome :-)